src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -44,11 +42,6 @@ public enum ArmPosition {
   private final AbsoluteEncoder armAbsoluteEncoder = armMotor.getAbsoluteEncoder(Type.kDutyCycle);
 
   public ScoringLocationUtil scoreLoc;
-  private ArmPosition desiredPosition = ArmPosition.STARTING;
```

Comment:
We should still be storing our desired position so we can reference it to see if we're at the desired position given a margin. This can be stored in goToPosition, for example.